examples/transit-gateway-cross-account-vpc-attachment/main.tf

```diff
@@ -100,6 +100,7 @@ resource "aws_ec2_transit_gateway_vpc_attachment" "example" {
 
 // ...and accept it in the first account.
 resource "aws_ec2_transit_gateway_vpc_attachment_accepter" "example" {
+  provider = "aws.first"
```

Comment:
Thanks for this fix, it looks like it landed earlier today due to an earlier pull request: #8972 and this pull request just needs to be rebased to include it 👍